conf/initializers/config.go

```diff
@@ -0,0 +1,19 @@
+package initializers
+
+import (
+	"fmt"
+
+	"go-crawler-challenge/helpers"
+
+	"github.com/beego/beego/v2/core/logs"
+	"github.com/beego/beego/v2/server/web"
+)
+
+func LoadAppConfig() {
+	configPath := fmt.Sprintf("%s/conf/app.conf", helpers.RootDir())
+	err := web.LoadAppConfig("ini", configPath)
+
+	if err != nil {
+		logs.Error(fmt.Sprintf("Load configuration failed: %v", err))
```

Comment:
@Lahphim Should we use `logs.Fatal` here? Because when the configuration file cannot be loaded the app won't be able to run 🤔  what do you think?